firebase-firestore/src/commonMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -34,14 +34,16 @@ public expect val Firebase.firestore: FirebaseFirestore
 /** Returns the [FirebaseFirestore] instance of a given [FirebaseApp]. */
 public expect fun Firebase.firestore(app: FirebaseApp): FirebaseFirestore
 
-public expect class NativeFirebaseFirestore
+internal expect class NativeFirebaseFirestore
 
 public class FirebaseFirestore internal constructor(private val wrapper: NativeFirebaseFirestoreWrapper) {
```

Comment:
Leaving comments to illustrate what I mean.

This class in common is the class users interact with. It is not an expect actual class and as such, to have the `android`/`ios`/`js` accessors, we need to write extension values on the platforms